rln/src/protocol.rs

```diff
@@ -25,13 +25,18 @@ use serde::{Deserialize, Serialize};
 use std::time::Instant;
 use thiserror::Error;
 
-pub use crate::utils::{add, bytes_to_field, mul, str_to_field, vec_to_field, vec_to_fr};
+pub use crate::utils::{
+    add, bytes_be_to_field, bytes_be_to_vec_field, bytes_be_to_vec_u8, bytes_le_to_field,
+    bytes_le_to_vec_field, bytes_le_to_vec_u8, field_to_bytes_be, field_to_bytes_le, mul,
+    str_to_field, vec_field_to_bytes_be, vec_field_to_bytes_le, vec_to_field, vec_to_fr,
+    vec_u8_to_bytes_be, vec_u8_to_bytes_le,
+};
 
 ///////////////////////////////////////////////////////
 // RLN Witness data structure and utility functions
 ///////////////////////////////////////////////////////
 
-#[derive(Debug, Deserialize, Clone)]
+#[derive(Debug, Serialize, Deserialize, Clone)]
```

Comment:
Seems like this does not have to be clone. Is clone necessary for something planned?
Also, the serde traits Serialize and Deserialize are not used.

Reply:
Again comes from some unsuccessful tests... removed in https://github.com/vacp2p/zerokit/commit/6f0dda79854455945eb53e2f79ab12568701be88